frontend/src/pages/shipments/ShipmentForm.tsx

```diff
@@ -125,22 +142,28 @@ const ShipmentForm: FunctionComponent<Props> = (props) => {
       />
       <div className="flex space-x-4">
         <SelectField
+          defaultValue={DEFAULT_MONTH}
           options={MONTH_OPTIONS}
           label="Label month"
           name="labelMonth"
           castAsNumber
           register={register}
+          registerOptions={{
+            validate: validateShipmentDate,
+          }}
           required
           errors={errors}
         />
-        <TextField
+        <SelectField
```

Comment:
The year is now a dropdown, which is a little quicker to fill out and easier to validate.

<img width="514" alt="Screen Shot 2021-05-03 at 4 29 50 PM" src="https://user-images.githubusercontent.com/3411183/116945567-db236300-ac2c-11eb-870c-f863a7cd202c.png">